packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/regionboundaries/RegionBoundariesFeature.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.regionboundaries;
+
+import android.annotation.TargetApi;
+import android.hardware.camera2.CaptureRequest;
+import android.os.Build;
+import android.util.Size;
+import io.flutter.plugins.camera.CameraProperties;
+import io.flutter.plugins.camera.features.CameraFeature;
+import java.util.Arrays;
+
+/**
+ * Holds the current region boundaries. When this is created, you must provide a
+ * CaptureRequestBuilder for which we can read the distortion correction settings from.
```

Comment:
Who are "you" and "we"?

Reply:
I have merged the `RegionBoundariesFeature` class with the `CameraRegions` class which makes this file obsolete. I made sure to include the feedback you provided in the new structure as well.

The `RegionBoundariesFeature` in fact was used as a factory to create an instance of the `CameraRegions` class. The actual implementation didn't represent a "Camera Feature" and was mis-used as a feature and consumed by other feature classes.